spec/models/provider_spec.rb

```diff
@@ -347,6 +347,19 @@
     end
   end
 
+  describe "training_providers" do
+    let(:accredited_provider) { create(:provider, accrediting_provider: "Y") }
+    let(:provider1) { create(:provider, accrediting_provider: "N") }
```

Comment:
This appears to be the default for this factory, setting `accrediting_provider` to `N`. If we want this to be explicit, how about a trait to mirror `accredited_body`?

Reply:
Fixed in 3c2e7a0. I've renamed those too as it's not quite as obvious without that param.